javax/src/main/java/org/jboss/ejb/protocol/remote/EJBServerChannel.java

```diff
@@ -138,6 +139,11 @@ final class EJBServerChannel {
             configuration.setObjectResolver(new ProtocolV3ObjectResolver(channel.getConnection(), true));
             configuration.setVersion(4);
         }
+        if (version < Protocol.JAKARTAEE_PROTOCOL_VERSION && Protocol.LATEST_VERSION >= Protocol.JAKARTAEE_PROTOCOL_VERSION) {
+            // EJB server uses EJB PROTOCOL version 4 or above but EJB client uses EJB PROTOCOL version 3 or below
+            // so in this case we need to translate classes from JavaEE API to JakartaEE API and vice versa
+            configuration.setClassNameTransformer(ClassNameTransformer.JAVAEE_TO_JAKARTAEE);
```

Comment:
Should we add a logging here to alert the user of this transformer? I think this is an important step, and a log will help debug if anything is not going as expected.